src/app/routes/cpsAsset/utils/getAdditionalPageData.js

```diff
@@ -0,0 +1,45 @@
+import isEmpty from 'ramda/src/isEmpty';
+import { STORY_PAGE } from '#app/routes/utils/pageTypes';
+import getAssetType from './getAssetType';
+import fetchPageData from '../../utils/fetchPageData';
+
+const pageTypeUrls = (assetType, service) => {
+  switch (assetType) {
+    case STORY_PAGE:
+      return [
+        {
+          name: 'mostRead',
+          path: `/${service}/mostread`,
+        },
+      ];
+    default:
+      return null;
+  }
+};
+
+const validateResponse = ({ status, json }, name) => {
+  if (status === 200 && !isEmpty(json)) {
+    return { [name]: json };
+  }
+
+  return null;
+};
+
+const fetchUrl = ({ name, path }) =>
+  fetchPageData(path).then(response => validateResponse(response, name));
+
+const getAdditionalPageData = async (pageData, service) => {
```

Comment:
I'm thinking the `variant` should be a parameter in this function

Reply:
I need to remove the variant param, as we are not using it yet. I will write a new issue to ensure we construct urls for variant based services too

Reply:
have update to remove the variant param for now

Reply:
@hindsc52 I believe you need to still remove variant from [here](https://github.com/bbc/simorgh/pull/6338/files/3046c6c3964309de9405a082d37f992e55f37adf#r419326543)

Reply:
Good spot @FK78 although I have now added variant back in as I am using a pre-existing util that has the variant logic baked in. This saves a small future PR